src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -371,6 +371,8 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
         impl_desc_type::ref,
     };
 
+    static std::mutex mutex;
+    std::unique_lock<std::mutex> lock(mutex);
```

Comment:
Modify static variable is not thread-safe, it makes sense to add lock.

Reply:
maybe we can use `std::call_once`?